pkg/fission-cli/util/util.go

```diff
@@ -535,7 +535,7 @@ func FunctionPodLogs(ctx context.Context, fnName, ns string, client cmd.Client)
 
 	if len(ns) == 0 {
 		ns = metav1.NamespaceDefault
-	} else if ns != metav1.NamespaceDefault {
+	} else {
 		podNs = ns
```

Comment:
@waitstory Please remove `podNs` variable from this function and use `ns` variable only. Environment and function should be in the same namespace.

Reply:
If possible, please make these additional changes in this PR:
- The selector used in this function will not work for `container` functions. Update the `FunctionPodLogs` function and make it work for container functions.
- Similar changes. Make [GetFunctionPodLogs](https://github.com/fission/fission/blob/main/pkg/fission-cli/logdb/kubernetes_log.go#L57) works for container functions.